src/main/java/edu/rosehulman/quota/controller/FreezeUserController.java

```diff
@@ -1,18 +1,17 @@
 package edu.rosehulman.quota.controller;
 
-import static spark.Spark.halt;
-
-import java.util.Optional;
-
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
-
 import edu.rosehulman.quota.Database;
 import edu.rosehulman.quota.model.User;
 import spark.Request;
 import spark.Response;
 import spark.Route;
 
+import java.util.Optional;
```

Comment:
What's the point of reorder? Or is this just the formatting...?

Reply:
That was intellij reordering them